<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Password_txt">
        <property name="text">
         <string>Encryption password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Decryption password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="decryptPasswordEdit"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Encrypt</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Decrypt_button">
       <property name="text">
        <string>Decrypt</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Explprer_button">
        <property name="text">
         <string>Explorer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Name_txt">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:700; font-style:italic;&quot;&gt; Encryption/Decryption&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::AutoText</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
